src/main/java/nl/tudelft/semgroup4/DashboardController.java

```diff
@@ -0,0 +1,96 @@
+package nl.tudelft.semgroup4;
+
+import nl.tudelft.model.Game;
+import nl.tudelft.model.Level;
+import nl.tudelft.model.Player;
+
+import org.newdawn.slick.geom.Rectangle;
+
+public class DashboardController {
+    private Game game;
+    private Dashboard dashboard;
+
+    /**
+     * Constructor of the Dashboard controller.
+     * 
+     * @param game
+     *            game of which the data is displayed
+     * @param dashboard
+     *            dashboard on which the controller makes changes
+     */
+    public DashboardController(Game game, Dashboard dashboard) {
+        this.game = game;
```

Comment:
You don't need to pass Game, because you're already passing Dashboard. You can then get the Game object through the Dashboard.